samples/custom_dispatch/cpu/plugin/standalone_plugin.c

```diff
@@ -0,0 +1,165 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// Demonstrates a standalone plugin exporting a single `import_mul` function.
+// This models kernel libraries and custom intrinsics where there's only
+// stateless functions. Standalone plugins can be compiled to portable ELFs that
```

Comment:
nit: grammar (also, this sample helps me understand how plugins fit into the broader architecture, thanks!)

```suggestion
// Demonstrates a standalone plugin exporting a single `import_mul` function.
// This models kernel libraries and custom intrinsics where there are only
// stateless functions. Standalone plugins can be compiled to portable ELFs that
```